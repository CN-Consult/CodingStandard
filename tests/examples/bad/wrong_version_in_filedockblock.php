<?php
/**
 *
 * @file
 * @version 220
 * @copyright 2020 CN-Consult GmbH
 * @author Daniel Haas <daniel.haas@cn-consult.eu>
 */
 