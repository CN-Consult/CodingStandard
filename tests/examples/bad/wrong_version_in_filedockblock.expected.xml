<?xml version="1.0" encoding="UTF-8"?>
<phpcs version="does not matter">
    <file name="examples/bad/wrong_version_in_filedockblock.php" errors="0" warnings="1" fixable="0">
        <warning line="5" column="4" source="..CNConsult_Sniffs_Commenting_FileComment.InvalidVersion" severity="5" fixable="0">Invalid version "220" in file comment; consider "0.1" or "0.1.1"    or "2.1" etc. instead</warning>
    </file>
</phpcs>
